### PR TITLE
added read and write payload functions

### PR DESCRIPTION
**Pull request by zhengmaxwell**

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Can you add a note to the change log for version 0.0.14
https://github.com/ramonaoptics/teensy-to-any

I meant to link to the file https://github.com/ramonaoptics/teensy-to-any/blob/master/CHANGELOG.md

**Comment by zhengmaxwell** [comment]
Why is it version 0.0.14 and not 0.0.11

**Comment by hmaarrfk** [comment]
Because I released version 0.0.13 without documentation.

You can add my release notes to the changelog too. THat would be helpful

https://github.com/ramonaoptics/teensy-to-any/releases

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### 0.0.14
+
+* Enable reading and writing a contiguous payload of bytes
+
```
Comment (hmaarrfk) [inline CHANGELOG.md:4]:
```suggestion

### 0.0.13

* Ensure that the system is compiled with I2C support for Teensy 3.2

### 0.0.12

* Preliminary Teensy 4.0 Support

### 0.0.11

* Added the ability to talk to 1 byte register address  I2C devices.
```

**Comment by hmaarrfk** [comment]
Thansk maxwell. I'm mergning and creating a tag.

**Pull request merged**